csrc/id_model/schedule.cpp

```diff
@@ -10,10 +10,48 @@
 
 namespace nvfuser {
 
+IterDomain* representativeId(const ValGroup& vg) {
```

Comment:
As a better engineering improvement, should we change (not in this PR) https://github.com/NVIDIA/Fuser/blob/2e293a32017d11aa872a7ec1a362e7fc9640f156/csrc/id_model/id_model.cpp#L743-L765
to use this function, or change `IdModel::addReplayAs` a whole as using utilities in schedule.{h,cpp}? cc @naoyam